src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -1036,18 +1036,18 @@ public Node visitTypeSwitchExpr(JsoniqParser.TypeSwitchExprContext ctx) {
     @Override
     public Node visitQuantifiedExpr(JsoniqParser.QuantifiedExprContext ctx) {
         List<QuantifiedExpressionVar> vars = new ArrayList<>();
-        QuantifiedExpression.QuantifiedOperators operator;
+        QuantifiedExpression.Quantification operator;
         Expression expression = (Expression) this.visitExprSingle(ctx.exprSingle());
         if (ctx.ev == null) {
```

Comment:
This is outside the scope of this PR but I just wanted to suggest that we also work on the names for variables used in the grammar at some point in the future.

Reply:
Yes, absolutely. With time we will keep cleaning up. Thank you for mentioning it.